dotcom-rendering/src/model/enhanceLists.ts

```diff
@@ -0,0 +1,49 @@
+import type { FEElement } from '../types/content';
+
+const enhance = (element: FEElement): FEElement[] => {
+	switch (element._type) {
+		case 'model.dotcomrendering.pageElements.ListBlockElement': {
+			switch (element.listElementType) {
+				case 'KeyTakeaways':
+					return [
+						{
+							_type: 'model.dotcomrendering.pageElements.KeyTakeawaysBlockElement',
+							keyTakeaways: element.items.flatMap(
+								({ title, list }) => {
+									if (title !== undefined) {
```

Comment:
perhaps a comment here to highlight if there is no title the element will be skipped/dropped/ignored?

Reply:
Good idea. Once we have validation set up properly in composer for this, I suspect we will disallow publishing a key takeaway with a missing title